external-crates/move/crates/move-unit-test/src/lib.rs

```diff
@@ -186,6 +187,7 @@ impl UnitTestingConfig {
         test_plan: TestPlan,
         native_function_table: Option<NativeFunctionTable>,
         cost_table: Option<CostTable>,
+        bytecode_deps_modules: Vec<CompiledModule>,
```

Comment:
This feels strange to me, and maybe should be something that is inside of TestPlan?

Reply:
Hmm `NamedCompiledModule` requires a `SourceMap` so seems a bit hacky to convert it like that. Is this safe?